src/child-applications/child-app-errors.js

```diff
@@ -21,7 +21,7 @@ function transformErr(ogErr, childApp) {
 	} else {
 		console.warn(`While ${childApp.status}, '${childApp.appLocation}' rejected its lifecycle function promise with a non-Error. This will cause stack traces to not be accurate.`);
 		try {
-			result = new Error(JSON.stringify(ogErr));
+			result = new Error(errPrefix + JSON.stringify(ogErr));
```

Comment:
errPrefix has the useful information that tells you which app is misbehaving and why